lib/string_additions.rb

```diff
@@ -0,0 +1,8 @@
+module StringAdditions
+  def remove_zero_width_chars
+    self.gsub(/^[\u200B-\u200D\uFEFF]/, '').gsub(/[\u200B-\u200D\uFEFF]$/, '') #Begins with and Ends with zero-width character
```

Comment:
This is looking good!  Do you think we need to handle `\u2060` as well? (https://en.wikipedia.org/wiki/Word_joiner)